recipes/cyrus-sasl/all/conanfile.py

```diff
@@ -77,36 +84,31 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        if self.options.with_openssl:
+        if is_msvc(self) or self.options.with_openssl:
             self.requires("openssl/[>=1.1 <4]")
-        if self.options.with_postgresql:
+        if self.options.get_safe("with_postgresql"):
             self.requires("libpq/15.4")
-        if self.options.with_mysql:
+        if self.options.get_safe("with_mysql"):
             self.requires("libmysqlclient/8.1.0")
-        if self.options.with_sqlite3:
+        if self.options.get_safe("with_sqlite3"):
             self.requires("sqlite3/3.44.2")
 
     def validate(self):
-        if self.settings.os == "Windows":
-            raise ConanInvalidConfiguration(
-                "Cyrus SASL package is not compatible with Windows yet."
-            )
```

Comment:
It was raising for Windows and this PR adds support for msvc. What about other compilers on Windows? It still doesn't work with MinGW AFAIK, it would be nice to not lost this information.

Reply:
I appreciate the concern, but I don't currently have a setup to test MinGW compatibility and I would not want to disable it just in case either. Feel free to open a PR.